### PR TITLE
add live anomaly reducer

### PR DESCRIPTION
**Pull request by ylwu-amzn**
*Issue #, if available:*

*Description of changes:*

* add live anomaly reducer

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: public/models/interfaces.ts [hunk 1]
```diff
@@ -102,6 +102,8 @@ export type Detector = {
   enabled?: boolean;
   enabledTime?: number;
   disabledTime?: number;
+  curState: DETECTOR_STATE;
+  initializationError: string;
```
Comment (yizheliu-amazon) [inline public/models/interfaces.ts:106]: ha, I guess those are for `Detector state page`. but not a blocker for approving it.

File: public/redux/reducers/liveAnomalyResults.ts [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+const DETECTOR_LIVE_RESULTS = 'ad/DETECTOR_LIVE_RESULTS';
+
+export interface Anomalies {
+  requesting: boolean;
+  totalLiveAnomalies: number;
+  liveAnomalies: Anomaly[];
+  errorMessage: string;
+}
+export const initialDetectorsState: Anomalies = {
```
Comment (yizheliu-amazon) [inline public/redux/reducers/liveAnomalyResults.ts:34]: rename to `initialDetectorLiveResults`?

File: public/models/interfaces.ts (hunk 1 above)
Comment (ylwu-amzn) [reply public/models/interfaces.ts:106]: Yeah, correct

File: public/redux/reducers/liveAnomalyResults.ts (hunk 2 above)
Comment (ylwu-amzn) [reply public/redux/reducers/liveAnomalyResults.ts:34]: sure, will change

**Pull request merged**